src/Helper/ValidateStreet.php

```diff
@@ -60,14 +60,15 @@ public static function validate(string $fullStreet, string $localCountry, ?strin
      */
     public static function getStreetRegexByCountry(string $local, string $destination): ?string
     {
-        if (
-            ($local === AbstractConsignment::CC_BE && $destination === AbstractConsignment::CC_NL) ||
-            ($local === AbstractConsignment::CC_NL && $destination === AbstractConsignment::CC_NL)
-        ) {
+        $localIsNlOrBe = in_array($local, [AbstractConsignment::CC_BE, AbstractConsignment::CC_NL]);
+        $destinationIsNl = $destination === AbstractConsignment::CC_NL;
+        $destinationIsBe = $destination === AbstractConsignment::CC_BE;
+
+        if ($localIsNlOrBe && $destinationIsNl) {
```

Comment:
Ik mis hier nog logica:

Van NL naar NL:
NL regex

Van BE naar NL:
NL regex

Van BE naar BE
BE regex

De rest moet null returnen. Je kan het zelf controleren in de backoffice.